README.md

```diff
@@ -32,25 +33,27 @@
 - Run the testing image: `make start`
 - Build the cpx components if necessary `make build-components`
 - Run one off test `make test`
-- Run and watch test `make test:watch`
-- Don't forget to stop the testing container when done `make stop`
-TODO: Remove node dependency and run from deno image.
+- Run and watch test `make test-watch`
+- Don't forget to stop the testing container when done `make stop` TODO: Remove
```

Comment:
Think this TODO should be on the next line still

Reply:
Fixed in [ac894a4](https://github.com/chapeaux/cpx-components/pull/21/commits/ac894a4ee20dec261e5b75add9f34c6b533df503)